### DISCUSSION TITLE
how to make haxeflixel read data files from a spritesheet made by adobe

### DISCUSSION THREAD

**Discussion by wormylmao** [HaxeFlixel/flixel#2375] [Q&A] [answered]
i need a simple answer pls cuz im just a starter

**Comment by RichardBray** [comment, marked as answer]
Hey, @wormylmao are you the same person who commented in this video?
https://www.youtube.com/watch?v=7jUKDSL2L6E&lc=Ugw-rEIy2k5bU4S8MyB4AaABAg

If you are, great to see you on the forums. If it's not you, please ignore my previous comment. 

I haven't used Adobe before but my understanding is that if you use the `fromSparrow` method in FlxAtlasFrames it should work.
https://api.haxeflixel.com/flixel/graphics/frames/FlxAtlasFrames.html#fromSparrow.

So something like this:
```hx
        // Helpers.hx
	public static function loadFrames(path: String): FlxFramesCollection {
		return FlxAtlasFrames.fromSparrow(
			'assets/images/$path.png',
			'assets/images/$path.json' // this could also be .XML file too
		);
	}
```

And then you can load it into your in your sprite like this:
```hx
        // Player.hx
	public function new(x: Float = 0, y: Float = 0) {
		super(x, y);
		frames = Helpers.loadFrames("characters/yeti");
```

NOTE: It doesn't have to be in the constructor.

**Reply by wormylmao** [reply]
thanks bro i needed this alot instead of using those dumb grid spritesheets
now i just need to find out how to move the character

**Reply by RichardBray** [reply]
Ha, glad I was able to help. Happy to help you move the character as well but I'll give you some time to figure that out first :)
